configure.ac

```diff
@@ -116,11 +116,14 @@ AC_CONFIG_FILES([opencl/extract_enqueues], [chmod +x opencl/extract_enqueues])
 AC_CONFIG_FILES([opencl/test_wrapper.sh], [chmod +x opencl/test_wrapper.sh])
 AC_CONFIG_FILES([ze/tracer_ze.sh], [chmod +x ze/tracer_ze.sh])
 AC_CONFIG_FILES([ze/test_wrapper.sh], [chmod +x ze/test_wrapper.sh])
```

Comment:
Can you remove the old one?

Reply:
The old ones are still used for X/tests/X_dummy_trace.yaml, to generate the dust from yaml.

Reply:
I see. I kind of don't like it but I don't see how we can improve it :')